### ISSUE TITLE
Befehl zur Erstellung des WebP Bildformates bricht nach kurzer Zeit ab

### ISSUE THREAD

**Issue by lukashillebrand** [FriendsOfShopware/FroshWebP#114]
### PHP Version

7.4.29

### Shopware Version

5.7.11

### Actual behaviour

Thumbnails wurden erfolgreich generiert, bis auf einige .tif Dateien mit:
`./bin/console sw:thumbnail:generate -f`

Dann wurde der Prozess wird abgebrochen

```
STACK: 4035
OFFSET: 0
  793/4035 [=====>----------------------]  19%media/image/Numo_1.jpg => Killed

  810/4035 [=====>----------------------]  20%media/image/Boogie_Classic01.jpg => Unable to launch a new process.
  812/4035 [=====>----------------------]  20%media/image/Foxter.jpg => Unable to launch a new process.
  870/4035 [======>---------------------]  21%Killed
```

### Expected behaviour

WebP Bildformat sollte aus den Shopware Bilder Alben generiert werden.

### Steps to Reproduce?

Konsolen Befehl ausführen:

`./bin/console frosh:webp:generate`

**Comment by shyim** [comment]
Profihost?

**Comment by lukashillebrand** [comment]
No, it's on Maxcluster.

**Comment by shyim** [comment]
Maybe they have some resource limitation as this runs very long 🤔

**Comment by lukashillebrand** [comment]
Yes this was an initial problem with the thumbnail generation, but then after updating the following MySQL configuration it worked:

```
 wait_timeout 14400
 max_allowed_package 1024M
```

maybe setting it even higher would work?
do you have any other suggestions?

**Comment by shyim** [comment]
Killed means some process on the system killed the job. It's nothing what can be setup by you

**Comment by lukashillebrand** [comment]
In the log I see this a lot at the time of script execution:

` server reached max_children setting (10), consider raising it`

**Comment by lukashillebrand** [comment]
After increasing a lot of PHP values:

https://capture.dropbox.com/d5YGcpJ9h8uNHpy0
